src/SwedbankPay.Sdk.Infrastructure/JsonSerialization/CustomLanguageConverter.cs

```diff
@@ -1,54 +1,54 @@
-﻿using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-
-using System;
-using System.Linq;
-
-namespace SwedbankPay.Sdk.JsonSerialization
-{
-    public class CustomLanguageConverter : JsonConverter
-    {
-        public override bool CanConvert(Type objectType)
-        {
-            return objectType == typeof(Language);
-        }
-
-
-        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
-        {
-            if (reader.TokenType == JsonToken.String)
-                return new Language((string)reader.Value);
-            if (reader.TokenType == JsonToken.StartObject)
-            {
-                var jo = JObject.Load(reader);
-                var language = jo.Values().FirstOrDefault()?.ToString();
-                return language != null ? new Language(language) : null;
-            }
-
-            if (reader.TokenType == JsonToken.Null)
-                return null;
-
-            throw new InvalidOperationException(
-                "Unhandled case for CustomLanguageConverter. Check to see if this converter has been applied to the wrong serialization type.");
-        }
-
-
-        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
-        {
-            if (null == value)
-            {
-                writer.WriteNull();
-                return;
-            }
-
-            if (value is Language language)
-            {
-                writer.WriteValue(language.ToString());
-                return;
-            }
-
-            throw new InvalidOperationException(
-                "Unhandled case for CustomLanguageConverter. Check to see if this converter has been applied to the wrong serialization type.");
-        }
-    }
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using System;
+using System.Linq;
+
+namespace SwedbankPay.Sdk.JsonSerialization
+{
+    public class CustomLanguageConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(Language);
+        }
+
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (reader.TokenType == JsonToken.String)

```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)